server/src/main/java/io/crate/metadata/settings/SessionSettings.java

```diff
@@ -94,6 +94,11 @@ public String applicationName() {
         return null;
     }
 
+    public String dateStyle() {
+        // Only available on coordinator.
```

Comment:
This should return the dataStyle, and the getter shouldn't be overriden in `CoordinatorSessionSettings`. see my other comment please.

Reply:
See comment further aboe. This is okay as it is and mirrors the applicationName handling